website/ref/language.md

```diff
@@ -109,15 +109,17 @@ represents Unicode codepoints in hexadecimal):
 
     -   `\t` is U+0009 HT (horizontal tabulation).
 
-    -   `\n` is U+000A LF (line feed), the standard line termination character
-        on Unix.
+    -   `\n` is U+000A LF (line feed, the standard line termination character on
+        Unix).
```

Comment:
The original formatting is fine; the "line feed" part serves to document what "LF" means.

Reply:
Okay, reverted. I made that change to move the "UNIX" text into the parenthetical because it only applies to UNIX like systems. But that is clearly outside the scope of this pull-request.